alpha/apps/kaltura/lib/batch2/bcdl/kReplacementHelper.php

```diff
@@ -278,6 +278,11 @@ public static function copyReadyReplacingEntryAssetToReplacedEntry($replacingEnt
 		$defaultThumbAssetNew = null;
 		$replacingEntry = $replacingEntryAsset->getentry();
 		$replacedEntry = entryPeer::retrieveByPK($replacingEntry->getReplacedEntryId());
+		if(!$replacingEntry || !$replacedEntry)
+		{
+			KalturaLog::info("One of the replacement entries is missing");
```

Comment:
Add move valid info here - replacing - exits or not , and replaced - exit of not,